pom.xml

```diff
@@ -29,7 +29,7 @@
     <parent>
         <groupId>org.apache</groupId>
         <artifactId>apache</artifactId>
-        <version>28</version>
+        <version>31</version>
```

Comment:
stupid question: what is this "apache" thing?

Reply:
> stupid question: what is this "apache" thing?

It is a parent pom with basic dependencies, which is the default for all Apache projects. I like their tooling such as the -Ppedantic build option searching for files without copy-right header! :-)
see https://maven.apache.org/pom/asf/ for details